rts/Lua/LuaSyncedRead.cpp

```diff
@@ -1943,6 +1944,33 @@ int LuaSyncedRead::GetTeamResourceStats(lua_State* L)
 }
 
 
+/***
+ *
+ * @function Spring.GetTeamDamageStats
+ * @number teamID
+ * @treturn damageDealt number
+ * @treturn damageReceived number
```

Comment:
Type goes in front but I can handle that.

Reply:
keelefi/spring#1

Reply:
Added to the PR branch, thanks!